composer.json

```diff
@@ -17,6 +17,7 @@
     "require": {
         "php": "^8.1",
         "ext-simplexml": "*",
+        "doctrine/annotations": "^2.0",
```

Comment:
I can't see any use of annotations... Do we really need to re-introduce this?

Reply:
This is used by the `VichUploaderExtension` class on function `registerAnnotationStrategy`. If not, a `MissingPackageException` is raised:

`Vich\UploaderBundle\Exception\MissingPackageException : Annotations support missing. Try running "composer require doctrine/annotations".`

Reply:
That's a conditional requirement, which is already covered in our "suggest" section. Please remove it from "require"

Reply:
OK.